packages/entando-apimanager/README.md

```diff
@@ -56,14 +56,12 @@ The store should contain both the `api` and `currentUser` reducers.
 
 `landingPage` is the callback used to redirect the user to the landing page when the `loginUser()` action is being used.
 
-### setApi({domain: null, useMocks: true, pathPrefix: ''})
+### setApi({domain: null, useMocks: true})
```

Comment:
Need to update also the documentation for the domain which now by default is ''

Reply:
done, thanks